lib/game.rb

```diff
@@ -0,0 +1,72 @@
+require_relative 'frame'
+
+class Game
+  attr_reader :frames
+  
+  def initialize
```

Comment:
Hi @tatsiana-makers. I feel like I had cracked dependency injection but now I seem to be lost again... Because the two classes were so tightly coupled, as soon as I began to mock the frame double methods, the tests seem become kind of redundant?  😖 Before I could check that a series of specific rolls were tallying up correctly, but now they don't? Am I missing something. Thank you!